roles/yoda_portal/defaults/main.yml

```diff
@@ -29,7 +29,7 @@ yoda_theme_mapping:
   wur: "Wageningen University & Research"
 
 # Yoda portal
-yoda_portal_version: "{{ yoda_version }}"
+yoda_portal_version: "portal-caching"
```

Comment:
Just adding a single comment as a reminder that this line will need to be removed prior to merging.